compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -66,11 +66,11 @@ struct ROCmOptions {
   void bindOptions(OptionsBinder &binder) {
     using namespace llvm;
     static cl::OptionCategory category("ROCm HAL Target");
-    binder.opt<std::string>("iree-rocm-target-chip", targetChip,
-                            cl::cat(category), cl::desc("ROCm target chip."));
+    binder.opt<std::string>("iree-hip-target", target, cl::cat(category),
+                            cl::desc("HIP LLVM target."));
     binder.opt<std::string>(
-        "iree-rocm-target-features", targetFeatures, cl::cat(category),
-        cl::desc("ROCm target features; e.g., '+sramecc,+xnack'."));
+        "iree-hip-target-features", targetFeatures, cl::cat(category),
+        cl::desc("HIP target features; e.g., '+sramecc,+xnack'."));
```

Comment:
Do we want to use "hip" here instead of "rocm"? Should the code be renamed from `ROCM/ROCMTarget.cpp` too?

Reply:
Yup we should rename the `ROCMTarget`. I have created https://github.com/iree-org/iree/issues/18224 and @nithinsubbiah  will do that later.

Reply:
(Created a separate task for it given it's largely disjoint and internal; compared to this one which is breaking)